src/ice_shelf/MOM_ice_shelf.F90

```diff
@@ -831,6 +926,47 @@ subroutine shelf_calc_flux(sfc_state_in, fluxes_in, Time, time_step_in, CS)
 
 end subroutine shelf_calc_flux
 
+subroutine integrate_over_ice_sheet_area(G, ISS, var, var_scale, var_out)
+  type(ocean_grid_type), intent(in) :: G  !< The grid structure used by the ice shelf.
+  type(ice_shelf_state), intent(in) :: ISS  !< A structure with elements that describe the ice-shelf state
+  real, dimension(SZI_(G),SZJ_(G)), intent(in)  :: var !< ice variable to integrate
```

Comment:
Please add units to the descriptions of these variables.  For `var`, `var_scale` and var_out`, I suspect that they would be `in arbitrary units [A ~> a]`, `[a A-1 ~> 1]`, and `in arbitrary units [a m2]`.  The description of `var_cell` is wrong, and I suspect that it should have units of `[a m2]`.